tests/test_on_data/test_recording_interfaces.py

```diff
@@ -409,6 +413,21 @@ def check_extracted_metadata(self, metadata: dict):
         assert metadata["NWBFile"]["session_start_time"] == datetime(2020, 11, 24, 15, 46, 56)
 
 
+class TestOpenEphysBinaryRecordingInterface_version_0_5_3_block_1_stream_1(
```

Comment:
I would prefer to name this test by purpose `TestOpenEphysBinaryWithBlocks` and add the provenance as a comment. But this is just my preference I see the argument for adding the provenance.